components/brave_wallet/renderer/js_ethereum_provider.cc

```diff
@@ -468,11 +408,10 @@ void JSEthereumProvider::SendAsync(gin::Arguments* args) {
           isolate, true));
 }
 
-v8::Local<v8::Value> JSEthereumProvider::IsConnected() {
-  v8::Isolate* isolate = v8::Isolate::GetCurrent();
+v8::Local<v8::Value> JSEthereumProvider::IsConnected(v8::Isolate* isolate) {
```

Comment:
Why don't we use `bool JSEthereumProvider::IsConnected(gin::Arguments* arguments)` so we can return true or false directly without constructing `v8::Boolean`? Same rule applies to `IsUnlocked`

Reply:
Use bool directly in https://github.com/brave/brave-core/pull/15498/commits/478e6a7cb3c335579e88d6b8ab275e306a874820. Left `IsUnlocked` unchanged as it needs to return Promise.